sda-commons-server-mongo-testing/src/main/java/org/sdase/commons/server/mongo/testing/MongoDb.java

```diff
@@ -59,6 +59,9 @@ public interface MongoDb {
   /** @return the MongoDB options String without leading question mark */
   String getOptions();
 
+  /** @return the MongoDB connection String */
+  String getConnectionString();
+
```

Comment:
This is just a bonus feature for the new type of test setup shown in `MorphiaBundleWithConnectionStringIT`, correct?

Reply:
Yes, correct.